src/main/java/net/sf/mzmine/modules/projectmethods/projectload/MyZIS.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2006-2015 The MZmine 2 Development Team
+ *
+ * This file is part of MZmine 2.
+ *
+ * MZmine 2 is free software; you can redistribute it and/or modify it under the
+ * terms of the GNU General Public License as published by the Free Software
+ * Foundation; either version 2 of the License, or (at your option) any later
+ * version.
+ *
+ * MZmine 2 is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+ * A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * MZmine 2; if not, write to the Free Software Foundation, Inc., 51 Franklin
+ * St, Fifth Floor, Boston, MA 02110-1301 USA
+ */
+
+package net.sf.mzmine.modules.projectmethods.projectload;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.zip.ZipInputStream;
+
+public class MyZIS extends ZipInputStream {

```

Comment:
I'd recommend a more descriptive name for this class - e.g. UncloseableZipStream or something on that line.. remember somebody might be reading your code after you and a line like

MyZIS zis = new MyZIS(); 

is not so easy to understand, so people have to dig into the class sources.